### DISCUSSION TITLE
Week 1 Thinking Cap

### DISCUSSION THREAD

**Discussion by yohman** [yohman/21S-DH151#13] [Assignments]
Find a web mapping site that excites you, and respond to the following prompts:

- In your class repo, create a new file in your Week1 folder titled `W1 Thinking Cap.md`
- Include a screenshot of the site
- Include a link to the site
- What is the site's agenda?
- Why does this website “speak” to you?
- How does the mapping work, and does it “work?” i.e., in your opinion, does it do what it’s supposed to do?
- What technologies does the map use? I.e. Google maps, leaflet, mapbox, etc?
- If you could change something, what would it be?

**Comment by lisa58** [comment]
https://github.com/lisa58/DH151/blob/main/Week1/W1%20Thinking%20Cap.md

**Comment by anncong** [comment]
https://github.com/anncong/DH151/blob/main/Week%201/Week%201%20Thinking%20Cap.md#week-1-thinking-cap

**Comment by kimjee8955** [comment]
https://github.com/kimjee8955/DH151/blob/main/Week1/Week%201%20Thinking%20Cap.md

**Comment by eliaranita1** [comment]
https://github.com/eliaranita1/DH151/blob/main/week%201/W1%20Thinking%20Cap.md

**Comment by derekwang99** [comment]
https://github.com/derekwang99/DH151/blob/main/Week%201/W1%20Thinking%20Cap.md

**Comment by JuliaWood1** [comment]
https://github.com/JuliaWood1/DH151/blob/main/Week1/W1%20Thinking%20Cap.md

**Comment by justisg** [comment]
https://github.com/justisg/DH151/blob/main/Week1/W1%20Thinking%20Cap.md

**Comment by adrianaromero819** [comment]
https://github.com/adrianaromero819/DH151/blob/main/Week%201/Week%201%20Thinking%20Cap.md

**Comment by alicelu170** [comment]
https://github.com/alicelu170/DH151/blob/main/Week%201/W1%20Thinking%20Cap.md

**Comment by LynnT2** [comment]
[https://github.com/LynnT2/DH151/blob/main/Week%201/W1%20Thinking%20Cap.md](https://github.com/LynnT2/DH151/blob/main/Week%201/W1%20Thinking%20Cap.md)

**Comment by lisalou36** [comment]
https://github.com/lisalou36/DH151/blob/main/week%201/week%201%20thinking%20cap.md

**Comment by ryanorlando** [comment]
https://github.com/ryanorlando/Dh151/blob/main/Week1/W1%20Thinking%20Cap.md

**Comment by MaryJaneHowland** [comment]
https://github.com/MaryJaneHowland/DH151/blob/main/Week%201/W1%20Thinking%20Cap%20Assignment.md

**Comment by hanarama** [comment]
https://github.com/hanarama/DH151/blob/main/Week%201/W1%20Thinking%20Cap.md

**Comment by yjwsb233** [comment]
https://github.com/yjwsb233/DH151/blob/main/Week1/Week%201%20Thinking%20Cap.md

**Comment by kaisanwatanabe** [comment]
https://github.com/kaisanwatanabe/DH151/blob/main/week%201/week%201%20thinking%20cap.md

**Comment by ivanma9** [comment]
https://github.com/ivanma9/DH151/blob/main/Week1/W1%20Thinking%20Cap.md

**Comment by zonaliao** [comment]
https://github.com/zonaliao/DH151/blob/ef408ce6af4f3ccf3606b3dc0a2fc41bd889a7df/Week1/W1%20Thinking%20Cap.md

**Comment by danielliuzy** [comment]
https://github.com/danielliu524/DH151/blob/main/Week1/W1%20Thinking%20Cap.md

**Comment by natgrace** [comment]
https://github.com/natgrace/DH151/blob/main/Week1/W1%20Thinking%20Cap.md

**Comment by jomh17** [comment]
https://github.com/jomh17/DH151/blob/main/WEEK%201/W1%20Thinking%20Cap.md

**Comment by alekhyavittalam** [comment]
https://github.com/alekhyavittalam/DH151/blob/main/Week1/W1%20Thinking%20Cap.md

**Comment by kmcinerny** [comment]
https://github.com/kmcinerny/DH151/blob/main/Week1/Week%201%20Thinking%20Cap.md

**Comment by ainsleighdouglas** [comment]
[https://github.com/ainsleighdouglas/DH-151/blob/main/Week1/W1%20Thinking%20Cap.md](url)

**Comment by ec028** [comment]
https://github.com/ec028/DH151/blob/main/Week1/W1ThinkingCap.md

**Comment by hoz-map** [comment]
https://github.com/hoz-map/DH151/blob/main/Week1/Week1%20Thinking%20Cap.md

**Comment by frida-barragan** [comment]
https://github.com/frida-barragan/DH151/blob/main/W1%20Thinking%20Cap.md

**Comment by sebdills** [comment]
https://github.com/sebdills/DH151/blob/73d82ffec126939142903cb5a1679c0901d40936/Week%201/W1%20Thinking%20Cap.md

**Comment by jmtanenbaum** [comment]
https://github.com/jmtanenbaum/DH151/blob/6ec3097897642080cbd59415ba68f727b1175929/week1/W1ThinkingCap.md

**Comment by michelle14lee** [comment]
https://github.com/michelle14lee/DH151/blob/main/Week1/W1%20Thinking%20Cap.md

**Comment by alexthegreat1** [comment]
https://github.com/alexthegreat1/DH151/blob/main/Week%201/W1%20Thinking%20Cap.md
